packages/sdk/src/types/DiscountVoucher.ts

```diff
@@ -40,6 +40,21 @@ interface SimpleProductDiscountUnit {
 	name: string
 }
 
+export interface DiscountUnitMultiple {
+	type: DiscountVouchersTypesEnum.UNIT
+	effect: 'ADD_MANY_ITEMS'
+	units: DiscountUnitMultipleOneUnit[]
+}
+
+interface DiscountUnitMultipleOneUnit {
```

Comment:
Export that interface too, because now dev will see this type inside "DiscountUnitMultiple" but he will not be able to refer to this directly